src/tenant/admin.py

```diff
@@ -119,14 +123,14 @@ def clean(self):
 
 class TenantAdmin(PublicSchemaOnlyAdminAccessMixin, admin.ModelAdmin):
     list_display = (
-        'schema_name', 'owner_full_name', 'owner_email', 'last_staff_login',
```

Comment:
The original owner name and email address should still be displayed with DEPRECATED in the label

Reply:
like this https://github.com/bytedeck/bytedeck/pull/1457/commits/1769ce9da98bf044fa999c38e1ba680880ba5eda ???

![Screenshot 2023-08-26 at 11 14 06 AM](https://github.com/bytedeck/bytedeck/assets/144783/f2bdc49a-714c-4cdc-aa12-a77455abfe48)